packages/vscode/src/index.ts

```diff
@@ -1,71 +1,103 @@
-import * as vscode from 'vscode';
-import * as lsp from 'vscode-languageclient/node.js';
+import { window, commands, workspace, ExtensionContext, TextDocument, Position } from 'vscode';
+import { LanguageClient, RequestType, TextDocumentPositionParams, ServerOptions, TransportKind } from 'vscode-languageclient/node';
+import { LanguageClientOptions } from 'vscode-languageclient';
```

Comment:
Changed the imports to importing specific things like this to match other files in the project